component/LinkableText/LinkableText.tsx

```diff
@@ -6,7 +6,7 @@ const LinkableText = ({text}: {text: string}) => {
 
     const textComponent = sanitizeNewLine?.map((textline, index) => {
         const textLineSplit = textline?.split(' ')
-        return <p className="font-normal text-sm" key={textline}>
+        return <p className="font-normal text-sm text-white" key={textline}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from the default text color to white might cause readability issues if the background color is light. Please ensure that this change doesn't affect the readability of the text across different themes or backgrounds.
```diff
- return <p className="font-normal text-sm" key={textline}>
+ return <p className="font-normal text-sm text-white" key={textline}>
```

Also, the logic for generating `hrefUrl` could potentially lead to incorrect URLs. If a word does not include 'http' or 'https', it's prepended with 'https://'. This might not be the desired behavior for all cases, especially when the word is not intended to be a URL.

Consider adding additional checks to ensure that the word is intended to be a URL before prepending 'https://'. For example, you could check if the word includes a '.' (dot), which is common in URLs. However, this is just a simple heuristic and may not cover all cases. A more robust solution would be to use a regular expression to validate the URL format.

```diff
- const hrefUrl = (word?.includes('http') || word?.includes('https')) ? word : `https://${word}`
+ const hrefUrl = (word?.includes('http') || word?.includes('https') || word.includes('.')) ? word : `https://${word}`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->